packages/in_app_purchase/in_app_purchase_ios/example/ios/RunnerTests/Stubs.h

```diff
@@ -54,6 +54,7 @@ API_AVAILABLE(ios(11.2), macos(10.13.2))
 @end
 
 @interface FIAPReceiptManagerStub : FIAPReceiptManager
+@property(assign, nonatomic) BOOL returnError;
```

Comment:
Needs a declaration comment. Test code is code that people need to understand and maintain like any other code, so has the same requirements on having comments explaining it.